lib/Models/Cesium3DTilesCatalogItem.ts

```diff
@@ -24,12 +24,10 @@ import { ItemSearchResult } from "./ItemSearchProvider";
 // A property name used for tagging a search result feature for highlighting/hiding.
 const SEARCH_RESULT_TAG = "terriajs_search_result";
 
-export default class Cesium3DTilesCatalogItem extends MappableMixin(
-  SearchableItemMixin(
-    FeatureInfoMixin(
-      Cesium3dTilesMixin(
-        CatalogMemberMixin(CreateModel(Cesium3DTilesCatalogItemTraits))
-      )
+export default class Cesium3DTilesCatalogItem extends SearchableItemMixin(
+  FeatureInfoMixin(
+    Cesium3dTilesMixin(
+      CatalogMemberMixin(CreateModel(Cesium3DTilesCatalogItemTraits))
```

Comment:
Isn't this missing MappableMixin now? Or is that part of a different mixin?

Reply:
`Cesium3dTilesMixin` includes `MappableMixin`